src/compiler/__tests__/computed.statement.import.test.ts

```diff
@@ -0,0 +1,111 @@
+  });
+
+  it(`import(b + '/' + c) should fail and return an error object`, () => {
+    // Tests the case:
+    // import(b + '/' + c);
+    const result = computedStatementToPath(getAst(`b + '/' + c`));
+    expect(result).toEqual({ error: `You're computed import needs to start with a string! i.e. './'` });
+  });
+
+  it('import(`./atoms/${a}`) should return a valid pattern', () => {
+    // Tests the case:
+    // import(`./atoms/${a}`);
+    const result = computedStatementToPath(getAst('`./atoms/${a}`'));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it('import(`./atoms/${b}/${c}`) should return a valid pattern', () => {
+    // Tests the case:
+    // import(`./atoms/${b}/${c}`);
+    const result = computedStatementToPath(getAst('`./atoms/${b}/${c}`'));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.DOUBLE_ASTERISK, '/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it('import(`./atoms/${b}${c}`) should return a valid pattern', () => {
+    // Tests the case:
+    // import(`./atoms/${b}${c}`);
+    const result = computedStatementToPath(getAst('`./atoms/${b}${c}`'));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it('import(`./atoms/${b}${c}/${d}`) should return a valid pattern', () => {
+    // Tests the case:
+    // import(`./atoms/${b}${c}`);
+    const result = computedStatementToPath(getAst('`./atoms/${b}${c}/${d}`'));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.DOUBLE_ASTERISK, '/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it('import(`./atoms/${b + \'/\' + c}`) should return a valid pattern', () => {
+    // Tests the case:
+    // import(`./atoms/${b + '/' + c}`);
+    const result = computedStatementToPath(getAst("`./atoms/${b + '/' + c}`"));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.DOUBLE_ASTERISK, '/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it('import(`./atoms/${b + \'/\' + c}`) should return a valid pattern', () => {
+    // Tests the case:
+    // import(`./atoms/${b + '/' + c}`);
+    const result = computedStatementToPath(getAst("`./atoms/${b + '/' + c}`"));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.DOUBLE_ASTERISK, '/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it(`import('./assets/' + var + '.svg');  should return a valid pattern`, () => {
+    // Tests the case:
+    // import('./assets/' + var + '.svg');
+    const result = computedStatementToPath(getAst(`'./assets/' + a + '.svg'`));
+    expect(result).toEqual(['./assets/', expressionValueTypes.DOUBLE_ASTERISK, '.svg']);
+  });
+
+  it(`import('./' + path + (() => 'test')); should return an error object`, () => {
+    // Tests the case:
+    // import('./' + path + (() => 'test'));
+    const result = computedStatementToPath(getAst(`'./' + path + (() => 'test')`));
+    expect(result).toEqual({ error: `Unsupported type provided to computed statement import` });

```

Comment:
There is one thing I don't like about it. Is that it can return both an array or an error. The interface should be 
```
{
  error?: string;
  paths ?: Array<string>  
}
```

Reply:
updated the code base. I was annoyed by it too